docs/frameworks/sveltekit.md

```diff
@@ -0,0 +1,229 @@
+---
+title: SvelteKit | Frameworks
+---
+
+# SvelteKit
+
+> For `Type declarations`, `Prompt for update` and `Periodic SW Updates` go to [Svelte](/frameworks/svelte.html) entry.
+> 
+
+You should remove all references to [SvelteKit service worker module](https://kit.svelte.dev/docs#modules-$service-worker) <outbound-link />
+to disable it on your application.
+
+Since `SvelteKit` will use `SSR / SSG`, we need to add the `ReloadPrompt` component using `dynamic import`. `Vite Plugin PWA`
+will only register the service worker on build, it is aligned with the current behavior of
+[SvelteKit service worker module](https://kit.svelte.dev/docs#modules-$service-worker) <outbound-link />.
+
+The best place to include the `ReloadPrompt` is on the main layout of the application:
+
+<details>
+  <summary><strong>src/routes/__layout.svelte</strong> code</summary>
+
+```html
+<script>
+  import { onMount } from 'svelte'
```

Comment:
Svelte files are indented with tabs by default, so it might make sense to use tabs here to make it easier for people to copy/paste